scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -326,6 +326,16 @@ class ExperimentalScanner(
         return nestedProvenanceScanResults
     }
 
+    private fun Collection<Package>.filterNotConcluded(): Collection<Package> =
```

Comment:
Ah, I overlooked this: Input types should be as generic as possible (`Collection`) but output types as specific as possible, so the return type should stay a `List`.

Reply:
That was deliberate, because if I change the input type to `Collection` I would have to call `toList()` on the input if `skipConcluded == false` which would copy the list to a new `ArrayList` for no reason. And it's a private function after all, I think for those it's totally fine to be tailored to the use case of the class, that's why I didn't use `Collection` in the first place.

Reply:
Ok, in this case I'll leave it to you whether to use `Collection` or `List` in both places. Approving in any case.